### ISSUE TITLE
Raspberry Pi freezes immediately after finishing scrapping with error msg

### ISSUE THREAD

**Issue by Seppo007** [RetroPie/EmulationStation#42]
Directly after ending the scraping-process inside Emulationstation the pi freezes.
Emulationstation aborts with the following error message:

emulationstation: /home/pi/RetroPie-Setup/tmp/build/emulationstation/es-core/erc/components/IList.h:124: const UserData& IList::getSelected() const [with EntryData = TextListData; UserData = FileData*]: Assertion 'size() > 0' failed.
Aborted

I can't acces the IList Header myself, it seems that the header is generated when the Scraper is starting to scrape.

The problem always occurs independently of the systems that are selected to scrape for and the scraping provider.

Also the integrated WLAN-module seems to be turned off or the whole system got catched up in an inifinite loop, because you even can't access the pi via SSH to reboot it.

So the only option is to hard reset the pi via unplugging the pi from the socket.
Had the same problem with Raspbian Wheezy and Pi2 now using Retropie 3.6 on Raspbian Jessie with Pi 3.

Problem occurs even if the scraping process is started and immediately stopped after.

**Comment by HerbFargus** [comment]
Out of curiosity have you installed kodi? Changing themes after installing Kodi gives the same error

https://github.com/RetroPie/EmulationStation/issues/37

I wonder if this is the same bug.

**Comment by joolswills** [comment]
It probably is.

**Comment by Seppo007** [comment]
Hi there, i'm currently rebuilding my whole system on another sd card starting with prebuild RetroPie 3.6 img. I will inform you at what stage of rebuilding the state of my main sd-card the problem will reoccur again. 

And thanks for the hint with kodi. I've installed it but I'm not quite sure if i tried to change the theme afterwards, will test this also.

**Comment by Seppo007** [comment]
so i finished my testing and rebuilding the state of my main sd-card to another. I've done it step by step, so i filled up the rom folders step by step, emulator for emulator. Then I used the emulationstation scraper for each emulator after this step and restartet emulationstation for the next roms i've uploaded then for another emulator.

So far none of my roms is causing this behaviour (could have been caused by signs inside the file name that couldn't be processed by the script, etc.).

At last i installed kodi. So i did nothing else after that, no changes in themes, didn't upload any new roms to scrap and so on. When I started the Scraper after this and finished the scraping (just scraped 3 - 5 games) i got the same error as mentioned before.

So Kodi itself in the state it's able to install for now is causing this error alone.
Hope that will help you getting rid of it.

Is there any possibility to remove kodi safely so it won't be used inside the scraping anymore?
